.github/workflows/commit_release_github_release.yaml

```diff
@@ -0,0 +1,59 @@
+name: build_commit_debug_f_d
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build_release_send_to_github_release:
+    name: ReleaseAPKtoGR
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
+          java-version: '11'
+      - uses: subosito/flutter-action@v2
+        with:
+          flutter-version: '3.0.5'
+          channel: 'stable'
+      - run: flutter --version
+      - name: Flutter upgrade
+        run: flutter upgrade
+      - name: AppKeys
+        run: |
+          touch secrets.json
+          echo "{" >> secrets.json
+          echo " \"apiKeyTRAKT\": \"${{secrets.API_KEY_TRAKT}}\"," >> secrets.json
+          echo " \"apiKeyOMDB\": \"${{secrets.API_KEY_OMDB}}\"," >> secrets.json
+          echo " \"apiKeyTMDB\": \"${{secrets.API_KEY_TMDB}}\"," >> secrets.json
+          echo " \"facebook_app_id\": \"${{secrets.FACEBOOK_APP_ID}}\"," >> secrets.json
+          echo " \"fb_login_protocol_scheme\": \"${{secrets.FACEBOOK_CLIENT_TOKEN}}\"," >> secrets.json
+          echo " \"facebook_client_token\": \"${{secrets.FB_LOGIN_PROTOCOL_SCHEME}}\" " >> secrets.json
+          echo "}" >> secrets.json
```

Comment:
up to u but u can save in secrets whole json and add it by one string

Reply:
Not good idea
I asked Sergey and we came to that case